README.md

```diff
@@ -7,7 +7,7 @@ Auditing for TLS certificates.
 
 ## Dependencies ##
 
- - [OpenSSL](https://www.openssl.org/source/), at least 1.0.0, preferably 1.0.2 (and up)
+ - [OpenSSL](https://www.openssl.org/source/), at least 1.0.0q, preferably 1.0.1l or 1.0.2 (and up)
```

Comment:
1.0.0q has the check we want? Hmm, I'll look into this, there might be a Ubuntu package available somewhere... (but don't wait for that)